### DISCUSSION TITLE
Question: How to publish project?

### DISCUSSION THREAD

**Discussion by Makstvell** [microsoft/aspire#1819] [Q&A] [answered]
Question: Can I publish project and start this project in own computer? 
If yes how I can do it? 
And If I have two projects asp.net. This projects uses own's appsettings and [Ip, port].. Can I see it at dashboard? 
And question I have running and finished statuses, can I restart project from dashboard?

for example :
<img width="726" alt="image" src="https://github.com/dotnet/aspire/assets/78256540/b097caad-4f4c-4084-85da-74e385094a50">

**Comment by davidfowl** [comment, marked as answer]
> Question: Can I publish project and start this project in own computer?If yes how I can do it?

Aspire has no tooling for you to deploy to your own computer. [Aspirate](https://github.com/prom3theu5/aspirational-manifests) is tooling that can deploy aspire apps to a Kubernetes cluster or spit out a docker compose file to run locally (or on a server with compose). The aspire orchestrator is for local development only and is not meant to be deployed anywhere. That said, the dashboard is now usable outside of the local experience in preview 3, but it still needs to be supported by various [deployment environments](https://github.com/prom3theu5/aspirational-manifests/issues/118).


I recommend watching this https://youtu.be/KEcUfMbCgpA?t=2749. It should provide some clarity (with diagrams).

> And question I have running and finished statuses, can I restart project from dashboard?

https://github.com/dotnet/aspire/issues/295

**Reply by Makstvell** [reply]
Ok, thanks.

**Reply by Makstvell** [reply]
I understood that I must change [ip:port] in launchsettings files of projects if I want that it show up at the dashboard. I am so sorry but how to create docker compose file? I found only how to deploy app to azure. But how I can deploy it to docker? Where I can find information about it?

**Reply by DamianEdwards** [reply]
Look at the Aspirate tool: https://github.com/prom3theu5/aspirational-manifests?tab=readme-ov-file#producing-manifests

**Reply by Makstvell** [reply]
Thanks
